tils/dupal-8-dynamic-author-box.mdx

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
@KashurDev Can you add screenshots of the view and also the output?

Reply:
<img width="1129" alt="Screenshot 2021-08-25 at 12 45 31 PM" src="https://user-images.githubusercontent.com/50568726/130745345-f86ee108-701c-4fc9-87dc-be92e3888e85.png">

<img width="1440" alt="Screenshot 2021-08-25 at 12 44 46 PM" src="https://user-images.githubusercontent.com/50568726/130745433-69230c52-efe7-47f9-a12f-ee3a6138502a.png">

Result:

<img width="845" alt="Screenshot 2021-08-25 at 12 43 54 PM" src="https://user-images.githubusercontent.com/50568726/130745463-64be5576-7b68-4de7-81d5-c346e76239bd.png">